scss/_patterns_pagination.scss

```diff
@@ -13,7 +13,8 @@
     }
   }
 
-  .p-pagination {
+  .p-pagination,
```

Comment:
I think we should by fine by doing `ul.p-pagination`, `ol.p-pagination` here to make sure it's only applied when needed.

But a comment in code should be added to mention that this is deprecated.